src/components/SourcesTree.js

```diff
@@ -237,7 +243,8 @@ module.exports = connect(
   state => {
     return {
       shownSource: getShownSource(state),
-      selectedSource: getSelectedSource(state)
+      selectedSource: getSelectedSource(state),
+      debuggeeUrl: getDebuggeeUrl(state)
```

Comment:
i see... lets stick to props and just compute this every time we need it. Also - is there a better name for the function to differentiate from the raw value